test/integration/simple-daemonset-annotated/00-assert.yaml

```diff
@@ -1,3 +1,12 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
can you change the install or assert of this test to make it fail? this way we can see what the collector does and document it here, after that we can revert the change 🥇